### PR TITLE
.julia to launch julia

### PR DESCRIPTION
**Pull request by tosalonijain**
I witnessed that to launch julia in my set up, whether through source code or the other way round, I need to run "./julia" and not "julia" simply. If in case its system dependent or its possible to launch using "julia" then please let me know. I suppose that then both the options should be mentioned in the doc.
Thank you.

### CODE REVIEW

**Comment by KristofferC** [comment]
Isn't the text above describing exactly what to do so you can run it using `julia`?

**Comment by KristofferC** [comment]
I think the text above describes how to set up your PATH so that it is possible running julia with just `julia` instead of `./julia`?

**Comment by tosalonijain** [comment]
@KristofferC , doesn't it say something else:
default

**Comment by KristofferC** [comment]
Yes, so 

> Once it is built, you can run the `julia` executable using its full path in the directory created above (the `julia` directory). 

Means that you can do `./julia`. But then it says:

> To run julia from anywhere you can:
> - add an alias (in bash: echo "alias julia='/path/to/install/folder/bin/julia'" >> ~/.bashrc && source ~/.bashrc), or
> - add a soft link to the julia executable in the julia directory to /usr/local/bin (or any suitable directory already in your path), or
> - add the julia directory to your executable path 

etc to explain what to do so you can just do `julia` and no longer need to put the full path. At least, that is how I interpret it.

**Comment by tosalonijain** [comment]
@KristofferC , Thank you, I got your point. But shall we also document the ./julia approach alongside, so the user can run julia after entering into the directory?

**Comment by StefanKarpinski** [comment]
This issue isn't really Julia-specific but it certainly wouldn't hurt to be a bit more explicit about this.

**Comment by tosalonijain** [comment]
@KristofferC  @StefanKarpinski shall the documents be amended a bit to fit both the ways? I shall send a PR if it seems correct to you people.
Thank you.

**Comment by ViralBShah** [comment]
Yes, it would be good to update this PR to do `./julia` after entering the directory, and leave the current approach as it is.

File: README.md [hunk 1]
```diff
@@ -99,7 +99,9 @@ If you need to build Julia on a machine without internet access, use `make -C de
 
 **Note:** The build process will fail badly if any of the build directory's parent directories have spaces or other shell meta-characters such as `$` or `:` in their names (this is due to a limitation in GNU make).
 
-Once it is built, you can run the `julia` executable using its full path in the directory created above (the `julia` directory). To run julia from anywhere you can:
+Once it is built, you can run the `julia` executable after you enter your julia directory and run
+    ./julia
```
Comment (yuyichao) [inline README.md]: Note that this is Unix only.

Comment (tosalonijain) [reply README.md]: @yuyichao Are you suggesting to specify that this is valid only for unix? I am sorry but could not clearly get your point. Could you please elaborate it a bit? 
Thank you

Comment (yuyichao) [reply README.md]: I'm just pointing out that this does not work on windows. You can either just mention that or give a way that works on windows too (e.g. `usr/bin/julia`/`usr/bin/julia-debug`) I don't know what people on windows prefer so I don't know what's better.

**Pull request merged**